lib/problems.rb

```diff
@@ -3,8 +3,51 @@
 # Time Complexity: ?
 # Space Complexity: ?
 def balanced(string)
-  raise NotImplementedError, "Not implemented yet"
+    
+    # define opening and closing elements
+    openers = ["{", "[", "("]
+    closers = ["}", "]", ")"]
+    
```

Comment:
I suggest using a hash here instead of arrays where the key is an open brace and the value is the close.